server/service/team_policies.go

```diff
@@ -502,6 +529,21 @@ func (svc *Service) modifyPolicy(ctx context.Context, teamID *uint, id uint, p f
 		}
 		policy.SoftwareInstallerID = softwareInstallerID
 	}
+	if p.ScriptID != nil { // indicates that policy ID is changing, but might be to 0 to remove
```

Comment:
nit: comment: script ID is changing

Reply:
Thx; resolved in [8396db9](https://github.com/fleetdm/fleet/pull/22472/commits/8396db900fe8459c54a82e9ff02a6e9f65617d49)